src/components/EventPreview/stories/EventPreview.stories.tsx

```diff
@@ -41,8 +77,8 @@ const stories = storiesOf('EventPreview', module)
       />
     ),
     {
-      info: {
-        maxPropObjectKeys: 10,
+      readme: {
+        content: hideType,
```

Comment:
Could we have only one example of hidden props and show examples with default fallbacks for other cases?